docs/README.md

```diff
@@ -0,0 +1,9 @@
+Some `*.md` files here are generated from their `*.template.md` counterpart. Generate them with:
```

Comment:
It's actually more complicated than that, because for example `overview.template.md` and usage-manual.template.md` contain

```
!INLINE ./start.md --hide-source-path
```

so even non-template files are used to generate other files ;) But I guess my sentence is enough for now to give the spirit.